test/unit/annotation_spec.js

```diff
@@ -70,12 +70,13 @@ describe('annotation', function() {
         { ref: annotationRef, data: annotationDict, }
       ]);
 
-      var annotation = AnnotationFactory.create(xref, annotationRef,
-                                                pdfManagerMock, idFactoryMock);
-      var data = annotation.data;
-      expect(data.annotationType).toEqual(AnnotationType.LINK);
+      AnnotationFactory.create(xref, annotationRef,
+        pdfManagerMock, idFactoryMock).then((annotation) => {
```

Comment:
Here and below: indent this line with two more spaces so it becomes:

```javascript
AnnotationFactory.create(xref, annotationRef,
    pdfManagerMock, idFactoryMock).then((annotation) => {
  var data = annotation.data;
  expect(data.annotationType).toEqual(AnnotationType.LINK);
  expect(data.id).toEqual('10R');
});

This is just our code style for multi-line statements so that we can keep the body indented with two spaces.
```